bundles/org.openhab.binding.pegelonline/README.md

```diff
@@ -0,0 +1,118 @@
+# PegelOnline Binding
+
+Binding to observe water level from german rivers. 
+Data is provided by german **Water-Route and Shipping Agency** [WSV](https://www.pegelonline.wsv.de/).
+Goal is to monitor actual water levels from rivers nearby your home. 
+In case of changing water levels the corresponding warning level is lowered or raised.
+
+## Supported Things
+
+| Label               | Description                                                                     | ID      |
+|---------------------|---------------------------------------------------------------------------------|---------|
+| Measurement Station | Station providing water level measurements                                      | station |
+
+
+## Discovery
+
+In case your home location coordinates are set the discovery will recognize all measurement stations within a radius of 50 km.
+Found Things are added in your Inbox.
+
+
+## Thing Configuration
+
+Thing configuration contains 3 sections
+
+* Station selection
+* Warning Levels of selected station
+* Refresh rate
+
+### Station selection
+
+Stations can be selected with an Universally Unique Identifier (uuid). 
+It's automatically added by the Discovery. 
+Configure a station manually with the [list of all available stations](https://www.pegelonline.wsv.de/webservices/rest-api/v2/stations.json) and choose the uuid of your desired measurement station.
+
+### Warning Levels
+
+<img align="right" src="./doc/Marburg.png" width="450" height="500"/>
+
+Each station has specific warning levels
+
+* Warning Levels 1 to 3
+* Flooding Levels frequent, occasionally and extreme
+
+Unfortunately these levels cannot be queried automatically. 
+Please select your [federal state](https://www.hochwasserzentralen.de/) and check if which levels they provide.
+The picture shows the levels of [measurement station Marburg of federal state Hesse](https://www.hlnug.de/static/pegel/wiskiweb2/stations/25830056/station.html?v=20210802152952)
+
+| configuration    | content   | unit | description               |
+|------------------|-----------|------|---------------------------|
+| uuid             | text      |  -   | Unique Station Identifier |
+| warningLevel1    | integer   |  cm  | Warning Level 1           |
+| warningLevel2    | integer   |  cm  | Warning Level 2           |
+| warningLevel3    | integer   |  cm  | Warning Level 3           |
+| hq10             | integer   |  cm  | Frequent Flooding Level   |
+| hq100            | integer   |  cm  | Century Flooding Level    |
```

Comment:
Assure this documentation, parameter and channel naming are aligned. Frequent, Decade, Century, Extreme etc.

Reply:
Changes aligned regarding decade and century flooding